news-search/src/App.js

```diff
@@ -1,29 +1,52 @@
+import { useEffect, useState } from 'react';
 import './App.css';
 
 import 'bootstrap/dist/css/bootstrap.min.css';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Axios from "axios";
 
-// import NavbarExpand from './components/Navbar'
 import Home from './pages/Home';
 import SharedLayout from './pages/SharedLayout';
-import Abortion from './pages/Abortion';
-import Bitcoin from './pages/Bitcoin';
-import GlobalWarming from './pages/GlobalWarming';
-import Inflation from './pages/Inflation'
 import Error from './pages/Error';
 
 function App() {
+  const [feed, setFeed] = useState([]);
+
+  useEffect(()=> {
+    allTopics();
+  },[]);
+
+  const allTopics = () => {
+    Axios.get("http://localhost:3001/getArticles").then((response)=> {
+      setFeed(response.data);
+    })
+  }
+
+  function changeTopic(topic) {
+    Axios.get(`http://localhost:3001/${topic}`).then((response)=> {
+      setFeed(response.data);
+    });
+  }
+
+  const changeTopicGlobalWarming = () => changeTopic('global-warming')
```

Comment:
May be easier in the future if we write a general arrow functions for such a case that takes a topic as a parameter?